dist/tools/commit-msg/check.sh

```diff
@@ -35,11 +35,16 @@ ERROR="$(git log \
         msg_length=$(echo "${msg}" | awk '{print length($0)}')
 
         if [ ${msg_length} -gt ${MSG_MAX_LENGTH} ]; then
-            echo "Commit message is longer than ${MSG_MAX_LENGTH} characters:" >&2
-            echo "    \"${msg}\"" >&2
+            WARNING=""
             if [ ${msg_length} -gt ${MSG_STRETCH_LENGTH} ]; then
+                MSG="Error: Commit message is longer than ${MSG_STRETCH_LENGTH} characters:"
                 echo "error"
```

Comment:
I think this has to go after the actual printout, but I'm not sure.

Reply:
I don't think so.  The messages goes to STDERR, the "error" goes to STDOUT and eventually to `ERROR`, which is then tested for being empty or not.

Reply:
You are right! Sorry, somehow overlooked the `>&2` for the message.